### PR TITLE
docs: misc

### PR DESCRIPTION
**Pull request by justinmk**
- docs: remove "#" comment char in `@return`

### CODE REVIEW

File: MAINTAIN.md [hunk 1]
```diff
@@ -136,36 +136,38 @@ Non-technical dependencies
...
+* Runner versions:
+    * For special-purpose jobs where the runner version doesn't really matter,
+      prefer `-latest` tags so we don't need to manually bump the versions. An
+      example of a special-purpose workflow is `labeler.yml`.
+    * For our testing jobs, which are in `test.yml` and `build.yml`, prefer to
+      use the latest stable (i.e. non-beta) version explicitly. Avoid using the
+      `-latest` tags here as it makes it difficult to determine from an
+      unrelated PR if a failure is due to the PR itself or due to GitHub bumping
+      the `-latest` tag without our knowledge. There's also a high risk that
+      automatic bumping the CI versions will fail due to manual work being
```
Comment (zeertzjq) [inline MAINTAIN.md:165]:
```suggestion
      automatically bumping the CI versions will fail due to manual work being
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -289,7 +289,8 @@ For convenience you can filter the regeneration by target (api, lua, lsp) using
 
 ### Lua docstrings
 
-Lua documentation uses a subset of [EmmyLua] annotations. See [:help dev-doc-lua][dev-doc-lua].
+Use [LuaLS] annotations in Lua docstrings to annotate parameter types, return
+types, etc. See [:help dev-doc-lua][dev-doc-lua].
```
Comment (justinmk) [inline CONTRIBUTING.md:293]: we're using luals annotations, not emmylua, right? @clason

**Comment by lewis6991** [comment]
I'm not sure this was a good change. `#` is used to deliminate the return type from the description. Without it, the first word is used as a name for the return value.

https://github.com/LuaLS/lua-language-server/wiki/Annotations#return

If you use LSP hover on a function, you can see the effect it has.

File: CONTRIBUTING.md (hunk 2 above)
Comment (lewis6991) [reply CONTRIBUTING.md:293]: Well, emmylua isn't really active anymore so I think LuaLS is redefining it.

See: https://github.com/LuaLS/lua-language-server/issues/980#issuecomment-1082841463

**Pull request merged**